src/mobu/github_config.py

```diff
@@ -0,0 +1,130 @@
+            " configuration page). For example:"
+            " https://github.com/organizations/lsst-sqre/settings/apps/mobu-ci-data-dev-lsst-cloud"
+        ),
+        validation_alias="MOBU_GITHUB_CI_APP_ID",
+        examples=[123456],
+    )
+
+    private_key: str = Field(
+        ...,
+        title="Github CI app private key",
+        description=(
+            "Generated when the GitHub app was set up. This should NOT be"
+            " base64 enocded, and will contain newlines. You can find this"
+            " in 1Password; check the Phalanx mobu values for more details."
+        ),
+        validation_alias="MOBU_GITHUB_CI_APP_PRIVATE_KEY",
+        examples=[
+            dedent("""
+            -----BEGIN RSA PRIVATE KEY-----
+            abc123MeowMeow456abc123MeowMeow456abc123MeowMeow456abc123MeowMeo
+            abc123MeowMeow456abc123MeowMeow456abc123MeowMeow456abc123MeowMeo
+            abc123MeowMeow456abc123MeowMeow456abc123MeowMeow456abc123MeowMeo
+            etc, etc
+            -----END RSA PRIVATE KEY-----
+        """)
+        ],
+    )
+
+    webhook_secret: str = Field(
+        ...,
+        title="Github CI app webhook secret",
+        description=(
+            "Generated when the GitHub app was set up. You can find this"
+            " in 1Password; check the Phalanx mobu values for more details."
+        ),
+        validation_alias="MOBU_GITHUB_CI_APP_WEBHOOK_SECRET",
+    )
+
+    users: list[User] = Field(
+        ...,
+        title="Environment users for CI jobs to run as.",
+        description=(
+            "Must be prefixed with 'bot-', like all mobu users. In "
+            " environments without Firestore, users have to be provisioned"
+            " by environment admins, and their usernames, uids, and guids must"
+            " be specified here. In environments with firestore, only "
+            " usernames need to be specified, but you still need to explicitly"
+            " specify as many users as needed to get the amount of concurrency"
+            " that you want."
+        ),
+    )
+
+    scopes: list[GafaelfawrScope] = Field(
+        ...,
+        title="Gafaelfawr Scopes",
+        description=(
+            "A list of Gafaelfawr scopes that will be granted to the"
+            " user when running notebooks for a GitHub CI app check."
+        ),
+    )
+
+    accepted_github_orgs: list[str] = Field(
+        [],
+        title="Allowed GitHub organizations.",
+        description=(
+            "Any webhook payload request from a repo in an organization not in"
+            " this list will get a 403 response."
+        ),
+    )
+
+    @field_validator("users")
+    @classmethod
+    def check_bot_user(cls, v: list[User]) -> list[User]:
+        bad = [u.username for u in v if not u.username.startswith("bot-")]
+        if any(bad):
+            raise ValueError(
+                f"All usernames must start with 'bot-'. These don't: {bad}"
+            )
+        return v
```

Comment:
Maybe we should just put this check in `User`, where we can just add a regex parameter to the `Field` definition of `username` and not have to write a validator? I think there was a time when we still had non-`bot-` users used in mobu in some environments, but I think we've gotten rid of those.

Reply:
Added `pattern=r"^bot-mobu"` to the User model.  There are no non-`bot-mobu` users in any of the autostart config in Phalanx.